### DISCUSSION TITLE
Help Better Understand Role Permission Differences

### DISCUSSION THREAD

**Discussion by dd-manderson** [burkybang/SuiteAdvanced-Feedback#123] [Ideas]
When using the Role Permission Differences, it would be helpful if there was a way to identify the differences in roles.  For example, highlight or hide higher or lower permissions than the comparing role.    When trying to level up or consolidate a role, I want to see what permissions I must add to make the role equal.

**Comment by burkybang** [comment]
I don't understand. Can you provide a screenshot of the feature you’re talking about?

**Comment by dd-manderson** [comment]
See the page at:

`/app/setup/diffroles.nl`

And then pick two different roles.  You should see the differences between the two roles.  I'm looking for a way to better understand the differences between the two roles without having to scan and look at each difference.

**Reply by burkybang** [reply]
I understand. I'll check it out!

**Comment by burkybang** [comment]
@dd-manderson Would icons like these help at all? I'm experimenting with some ideas.

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/555d0c25-02db-4960-8e53-b9460b87a081)

**Reply by dd-manderson** [reply]
This looks great.   Maybe the icons can be a bit smaller; I'm sure everyone has their own preference.

One feature that would be super helpful is when you are comparing two roles (I didn't even know you could do more than one comparison!) is to show what changes need to be made to "level up" the other role.  So if we were looking at Buyer and 11: A/R Clerk and we wanted to make it show that 11: A/R Clerk was a super set of Buyer, you would hide all rows except the rows that had a red down arrow.  This would then tell me that I needed to add those specific permissions to 11: A/R Clerk to have parity and above with the Buyer.  Make sense?

Thanks for developing this tool.  It has a meaningful impact on my work every day!

**Reply by burkybang** [reply]
The arrow icons are in the latest release now but with tamer icons and not as big per your suggestion. Let me know what you think of it now.

I may create the second idea you have at some point. Any suggestion for the button label?

**Reply by dd-manderson** [reply]
Icons look great.  I was going to suggest to just use what was in Netsuite.

Not sure on the button name.   I think once I saw it in use it may become obvious.
